### PR TITLE
[tests] use AgentCheckTest for couch.py

### PR DESCRIPTION
**Pull request by degemer**
Couch test can now be run using the cache (previously it wasn't possible because
of missing environment variables)

### CODE REVIEW

File: ci/couchdb.rb [hunk 1]
```diff
@@ -60,11 +60,17 @@ def couchdb_rootdir
...
+        # Still needed to start
+        ENV['PATH'] = "#{couchdb_rootdir}/bin:#{ENV['PATH']}"
+        ENV['LD_LIBRARY_PATH'] = "#{couchdb_rootdir}/lib:#{ENV['LD_LIBRARY_PATH']}"
+        ENV['DYLD_LIBRARY_PATH'] = "#{couchdb_rootdir}/lib:#{ENV['DYLD_LIBRARY_PATH']}"
       end
     end
 
     task :before_script => ['ci:common:before_script'] do
       sh %(#{couchdb_rootdir}/bin/couchdb -b)
+      sleep 2
```
Comment (LeoCavaille) [inline ci/couchdb.rb]: Use `sleep_for` it shows nicer in the logs

File: tests/test_couch.py [hunk 2]
```diff
@@ -1,54 +1,44 @@
...
-        cr_svc_check = service_checks[1]
-        self.assertEqual(cr_svc_check['check'], self.check.SERVICE_CHECK_NAME)
-        self.assertEqual(cr_svc_check['status'], AgentCheck.CRITICAL)
-
+        self.run_check(self.config)
+
+        self.assertServiceCheck(self.check.SERVICE_CHECK_NAME,
+                                status=AgentCheck.OK,
+                                tags=['instance:http://localhost:5984'],
+                                count=1)
```
Comment (LeoCavaille) [inline tests/test_couch.py]: Nice. Can you throw a `self.coverage_report()` in the mix? It fails the test if you run the CI with `COVERAGE=true` and you didn't test all the metrics / service checks, I plan on making it a bit nicer but run it in the CI at some point.

**Comment by degemer** [comment]
Thanks @LeoCavaille, updated the check and ci.

**Comment by LeoCavaille** [comment]
Merging it, even if you broke the rabbitmq flavor :wink:

**Pull request merged**